### PR TITLE
[solidjs-pod] incorporated ream me view component

### PR DESCRIPTION
**Pull request by AllStackDev1**
Closes: https://github.com/thisdot/starter.dev-github-showcases/issues/943

image

# Create single repo readme view

## Background

As a user, by default when I navigate to a single repository I land on the code tab. Here I can see the folders and files found in the root directory of the repo and the name of the root branch. I can also see the repo’s readme file below the folder tree.

## Acceptance

- [ ] show root readme.md file with list icon next to the title

### CODE REVIEW

File: solidjs-tailwind/src/contexts/RepoContext.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  });
+  
+
+  const repo = [
+    readme
+  ]
+
+  return (
+    <RepoContext.Provider value={repo}>
+      {props.children}
```
Comment (hdJerry) [inline solidjs-tailwind/src/contexts/RepoContext.jsx:20]: try use the children solidjs provides use with.
https://www.solidjs.com/tutorial/props_children

File: solidjs-tailwind/src/components/RepoReadMe/RepoReadMe.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          {readme().text ? (
+            <>
+              <div class={styles.container} data-testid="readme">
+                <header class={styles.header}>
+                  <span class={styles.tocIconContainer}>
+                    <TOCIcon class={styles.tocIcon} />
+                  </span>
+                  <span class={styles.filename}>README.md</span>
+                </header>
+                <article class="prose py-6 px-10 max-w-none prose-headings:font-medium prose-p:my-3 prose-p:leading-6 prose-headings:border-b prose-headings:my-1 prose-headings:pb-2 prose-h1:my-4 prose-h2:my-4 prose-h3:border-none prose-h4:border-none prose-h5:border-none prose-img:inline prose-img:m-0 prose-a:text-blue-600 prose-a:no-underline prose-a:font-normal hover:prose-a:underline prose-li:my-0.5 prose-pre:bg-gray-100  prose-code:text-gray-800 text-gray-800 prose-code:px-1 prose-code:py-0.5 prose-code:rounded-md prose-code:bg-gray-100">
```
Comment (hdJerry) [inline solidjs-tailwind/src/components/RepoReadMe/RepoReadMe.jsx:30]: I thought we agreed on using modules.

**Review by hdJerry — CHANGES_REQUESTED** [review]
Also, the readme doesn't display as it's meant to, it just renders the text as it is.

File: solidjs-tailwind/src/contexts/RepoContext.jsx (hunk 1 above)
Comment (AllStackDev1) [reply solidjs-tailwind/src/contexts/RepoContext.jsx:20]: I don't think it is possible in the context, as i get this error below;

image

Also, this method of context I used was from their example guide

**Review by SDaian — APPROVED** [review]
LGTM

**Pull request merged**